eng/Versions.props

```diff
@@ -26,15 +26,8 @@
     imported. This OK because we want to just have an obvious salt for a local build.
   -->
   <PropertyGroup>
-    <!-- 
-      If this is developer build - we need to have a LARGER number than our official builds, which will
-      look like: 16.0.20181207.3
+    <VsixVersionPrefix>16.1.0</VsixVersionPrefix>
```

Comment:
Nice that we could remove everything else.

Reply:
yeah, arcade guys hoooooooked us up 👍